source/includes/api/v3/_campaigns.md

```diff
@@ -629,3 +629,303 @@ skip | true | Data offset index. Useful for pagination.
 sort | true | Sort column. Ex.: `["id","ASC"]`
 
 Filter Example: `/data/organizationId eq "eb4dfe83-f1fd-46dd-a69d-b7cf7b566319" and /data/status eq "complete"`
+
+
+## The Contact Household Campaign document
+
+```json
+{
+    "id": "00001820-1329-490c-b07c-2071c29329de",
+    "data": {
+        "contacted": false,
+        "campaignId": "c92162c6-f604-4caf-a964-d668d8bf67c8",
+        "successful": false,
+        "reservation": {
+            "reserved": false
+        },
+        "availability": {
+            "available": false,
+            "unavailableReason": null
+        },
+        "contactHouseholdId": "b66e4452-5fcd-4b79-bb1c-068ed7353192",
+        "state": "not-contacted-normal",
+        "statusId": "54aa7cef-877a-44ec-b2c5-442b43f5d758"
+    },
+    "meta": {
+        "etag": "295-1n/IqPXKKnZp0KmvaPoq7YWAIUI",
+        "created": "2019-09-05T20:54:33.580Z",
+        "modified": "2019-09-05T20:54:33.580Z",
+        "resource": "contact-household-campaigns",
+        "createdBy": "f7db3e84-e0f0-4b8b-94b2-2e5675ddddaa",
+        "isDeleted": false,
+        "modifiedBy": "f7db3e84-e0f0-4b8b-94b2-2e5675ddddaa"
+    },
+    "customerId": "bd4b51e6-804c-497d-8f60-cf7b12c56e06",
+    "securityGroupId": "2e61dbbc-02f2-4084-b36e-c3d1d39ed495"
+}
+```
+
+Attribute | Required? | Description
+--------- | --------- | -----------
+id | true | Unique identifier for the object.
+customerId | true | Customer this resource belongs to.
+securityGroupId | true | Security group of this resource.
+data | true | Campaign data.
+
+### data
+
+Attribute | Required? | Description
+--------- | --------- | -----------
+contacted | false | Flag indicating if the contact household has ever been contacted
```

Comment:
maybe "Flag indicating if the contact household has ever been contacted within the context of this campaign"